src/analytics/config.py

```diff
@@ -11,19 +11,28 @@ class Config:
     SECRET_KEY = 'sample-code-local-environment'
     #SERVER_PORT = 5000
 
+    CLARITY_API_BASE_URL= "https://clarity-data-api.clarity.io/v1/"
```

Comment:
We need to be aware of the security aspects of our system. We could explore the usage of environmental variables.